### ISSUE TITLE
[ACCESSIBILITY] All horizontal scrolling is not accessible (multiple areas of app affected by this)

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1455]
**Describe the accessibility issue**
The Previous/Next controls should not be located on each individual Twitter feed card. The screen reader user has to swipe 14 times just to reach the first previous control, and 15 to get to the first "next" control. 
Additionally, these repeating on multiple cards is going to cause an additional accessibility issue with the ID's for the Previous/Next not being unique. These Previous/next controls also need disabled/enabled states, as well as Pressed and Focus states.

**Summary of requirements for Twitter controls:**

- Controls should be separate from the individual twitter cards, (Perhaps at top of feed as in example from Precision planting)
- Controls must have disabled & enabled state
- Controls must have a pressed state 
- Controls must have a focus state 
- Controls must meet the Color contrast requirements for [WCAG 1.1.1 Non-text contrast](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) and [WCAG 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast) 

**To Reproduce
Steps to reproduce the behavior:**

1. Go to phone settings
2. Turn on screen reader like TalkBack or voiceover
3. Navigate down App home screen until you reach the Twitter feed
4. Attempt to get to next card without swiping

**Expected behavior:**
More like Precisionplanting twitter feed in example attached.

**Actual behavior:**
See IL app twitter feed w screen reader attached. 

**Screenshots**
Please see the videos attached. 
1 - is a good example of a twitter feed that is accessible for screen reader users

https://user-images.githubusercontent.com/82892873/158216918-53e07588-056d-41e5-9d2d-1503987565a7.mp4

2 - the IL app twitter experience for screen reader user

https://user-images.githubusercontent.com/82892873/158216979-3866b81f-197d-4369-a3b0-5060a3a436cf.mp4

**Smartphone (please complete the following information):**
 iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Comment by TodorBachvarov** [comment]
@acovher2 could you test the Twitter widget that I've updated? I'll need updated description (not just title) or it will be even better if you create new ticket. If the moved buttons in the Twitter Widget seems to be acceptable (for this specific widget), could we close this issue (since it was originally pointing to this specific widget). Since the accessibility issues will require different approach for every different panel/widget/element we will need separate issues.

**Comment by acovher2** [comment]
closing as This will be in a new all encompassing issue
